dist/svg.js

```diff
@@ -1694,20 +1704,53 @@ SVG.FX = SVG.invent({
 
       // convert current time to position
       if(!ignoreTime) this.pos = this.timeToPos(+new Date)
+      else {
+        // If the method at was called with a position above 1, we set it back to 1
+        if(this.pos > 1) this.pos = 1
+        if(this.situation.reversed) this.pos = 1 - this.pos
+      }
 
-      if(this.pos >= 1 && (this.situation.loop === true || (typeof this.situation.loop == 'number' && --this.situation.loop))){
+      if(this.pos < 0) this.pos = 0
+      if(this.pos >= 1){
```

Comment:
Why ignoreTime has to be true to cap the value to 1? Imo user can just call `at(1.3)` and it should start the next loop or cap to `1`. Thats why I handled the loop stuff first and then capped the values. I admit that I cannot fully follow your code.
Shouldnt the flow be like this?:

```
calculate position
correct position to reverse if needed
handle overflowing positions (say do the loop stuff)
now cap the values to [0,1]
other stuff...
```

Correct me if i am wrong but for me it looks a bit overcomplicated.